Library/Homebrew/cmd/audit.rb

```diff
@@ -878,12 +878,12 @@ def audit_line(line, lineno)
         problem "`#{$1}` is now unnecessary"
       end
 
-      if line =~ /#\{share\}\/#{formula.name}/
+      if line =~ %r{#\{share\}/#{formula.name}[/'"]}
         problem "Use \#{pkgshare} instead of \#{share}/#{formula.name}"
       end
 
-      if line =~ /share\/"#{formula.name}"/
-        problem "Use pkgshare instead of (share/\"#{formula.name}\")"
```

Comment:
No I thought it was better to use a more generic message. Do you think people would use `(pkgshare)`? I can take back the previous wording if you prefer.